### PR TITLE
Both Review panels can be now read-only

### PR DESCRIPTION
**Pull request by rkzel**
A bool variable has been introduced, so links in Activity log
can modify default behavior - which is to show review form
in those panels (unless review had already taken place).

No cleanup was intended here - there is another PR dealing
with Status and Avatar components. This PR is to strictly
add RO mode to the panels.

Screen (work submission, application submission looks pretty much the same):
Screenshot from 2019-12-10 16-25-04

### CODE REVIEW

File: apps/projects/app/components/Panel/ReviewApplication/ReviewApplication.js [hunk 1]
```diff
@@ -158,7 +158,8 @@ const ReviewApplication = ({ issue, requestIndex }) => {
             {request.review.feedback.length ? request.review.feedback : 'No feedback was provided'}
           </Text.Block>
         </React.Fragment>
-      ) : (
+      )}
+      {!(readOnly || 'review' in request) && (
```
Comment (chadoh) [inline apps/projects/app/components/Panel/ReviewApplication/ReviewApplication.js]: I would find this a lot easier to read if we got rid of the parentheses and did something like:

```js
!readOnly && !request.review
```

File: apps/projects/app/components/Content/IssueDetail/EventsCard.js [hunk 2]
```diff
@@ -53,13 +53,13 @@ IssueEvent.propTypes = {
 
 const applicationLink = (user, onReviewApplication, issue, index) => (
   <React.Fragment>
-    {user} submitted <Link onClick={() => onReviewApplication(issue, index)}>an application for review</Link>
+    {user} submitted <Link onClick={() => onReviewApplication(issue, index, true)}>an application for review</Link>
```
Comment (chadoh) [inline apps/projects/app/components/Content/IssueDetail/EventsCard.js]: I find it really hard to understand what a function will do with a raw boolean like this, and would prefer that we passed it with a name. We can do that like this:

```js
onReviewApplication({ issue, index, readOnly: true })
```

File: apps/projects/app/components/Panel/ReviewWork/ReviewWork.js [hunk 3]
```diff
@@ -155,7 +156,8 @@ const ReviewWork = ({ issue }) => {
             </Text.Block>
           </FieldText>
         </React.Fragment>
-      ) : (
+      )}
+      {!(readOnly || 'review' in work) && (
```
Comment (chadoh) [inline apps/projects/app/components/Panel/ReviewWork/ReviewWork.js]: Same comment here about making boolean logic readable

File: apps/projects/app/components/Panel/ReviewApplication/ReviewApplication.js (hunk 1 above)
Comment (rkzel) [reply apps/projects/app/components/Panel/ReviewApplication/ReviewApplication.js]: Fixed!

File: apps/projects/app/components/Panel/ReviewWork/ReviewWork.js (hunk 3 above)
Comment (rkzel) [reply apps/projects/app/components/Panel/ReviewWork/ReviewWork.js]: Fixed as well. Much more readable now.

File: apps/projects/app/components/Content/IssueDetail/EventsCard.js (hunk 2 above)
Comment (rkzel) [reply apps/projects/app/components/Content/IssueDetail/EventsCard.js]: Changed in all places for both panels. Actually easier than it seemed.

**Pull request merged**